src/messages.ts

```diff
@@ -280,15 +280,15 @@ export function handleMessage(this: any, msg: { data: string }): void {
 
       const configuration = this._configurations.get(watchedAddress)
 
-      const configurationEmitterResult =
-        configuration &&
-        configuration.emitter &&
-        configuration.emitter.emit(newState)
+      const emitterResult =
+        configuration && configuration.emitter
```

Comment:
If we don't have a `configuration` or `configuration.emitter` then don't call `emit` and leave it out of the `emitterResult`